src/H5P.c

```diff
@@ -788,7 +788,7 @@ H5Pget_size(hid_t id, const char *name, size_t *size /*out*/)
     The encoding of property values will be done according to the file format
     setting in fapl_id.
  USAGE
-    herr_t H5Pencode(plist_id, buf, nalloc, fapl_id)
+    herr_t H5Pencode2(plist_id, buf, nalloc, fapl_id)
```

Comment:
Actually the original is not functionally incorrect - the version macros will choose the correct procedure based on the arguments. Adding the "2" is necessary under certain conditions as we encounter from time to time in the tests.